app.py

```diff
@@ -0,0 +1,34 @@
+from flask import Flask, jsonify, request
+
+app = Flask(__name__)
+
+events = [
+    {
+        "dates": [
+            {
+                "day": 4,
+                "month": 2,
+                "times": {
+                    "1": {
+                        "end": 12,
+                        "start": 10
+                    },
+                    "2": {
+                        "end": 23,
+                        "start": 22
+                    }
+                },
+                "year": 2020
+            }
+        ],
+        "description": "first description",
+        "title": "event 1"
+    }
+]
+
+
+@app.route("/events/new", methods=["POST"])
+def create_event():
+    data = request.json
+    events.append(data)
+    return jsonify(events), 201
```

Comment:
Returning all the events is not a good pattern. Either return the created object or nothing.
More info:
* https://softwareengineering.stackexchange.com/questions/314066/restful-api-should-i-be-returning-the-object-that-was-created-updated
* https://stackoverflow.com/questions/19199872/best-practice-for-restful-post-response